sync/class.jetpack-sync-client.php

```diff
@@ -604,6 +631,52 @@ function filter_post_content_and_add_links( $post ) {
 		return $post;
 	}
 
+
+	function expand_wp_comment_status_change( $args ) {
+		return array( $args[0], $this->filter_comment_and_add_hc_meta( $args[1] ) );
+	}
+
+	function expand_wp_insert_comment( $args ) {
+		return array( $args[0], $this->filter_comment_and_add_hc_meta( $args[1] ) );
+	}
+
+	function filter_comment_and_add_hc_meta( $comment ) {
```

Comment:
This method should be renamed, as per other comments, to `filter_comment` (we don't add hc meta now)